spec/models/medical_history_spec.rb

```diff
@@ -47,4 +47,34 @@
       expect(patient.medical_history.indicates_risk?).to eq(true)
     end
   end
+
+  describe '#indicates_hypertension_risk?' do
+    it 'returns true if there was a prior heart attack' do
+      patient = create(:patient)
+      create(:medical_history, prior_heart_attack_boolean: true, patient: patient)
+
+      expect(patient.medical_history.indicates_hypertension_risk?).to eq(true)
```

Comment:
Did we not use the fancy rspec boolean helper to do this like we've done [here](https://github.com/simpledotorg/simple-server/pull/759/files#diff-fcb8e53799332de9bb6227ebcb0cde53R39) because it'd read weird like `be_indicates_hypertension_risk`?

Reply:
Haha yup exactly 😄 I like the magic helper because the tests read easier. In this case, since the test reads worse, I've avoided it.